aten/src/ATen/native/native_functions.yaml

```diff
@@ -1535,6 +1535,8 @@
 
 - func: copy(Tensor self, Tensor src, bool non_blocking=False) -> Tensor
   variants: function
+  dispatch:
+    CompositeExplicitAutogradNonFunctional: copy
```

Comment:
I remember a while back we were debating whether or not to make `copy()` a primitive, and we settled on "we can represent copy() with existing operators, so we shouldn't bother adding a new primitive unnecessarily".

I'm mostly just wondering with that in the background - what's the idea behind making it a primitive now? Now that we care about accurate striding, representing the decomp accurately is sufficiently more annoying that it would be easier for inductor to lower it directly?

Reply:
It is very annoying to write a decomposition that is both (1) functional and (2) preserves striding exactly.

Reply:
Can definitely believe that :)